### DISCUSSION TITLE
Multiplayer Games

### DISCUSSION THREAD

**Discussion by bettywolfy** [nesbox/TIC-80#1691] [Ideas]
well, why not support multiplayer games?
think about the possibilities... it would be amazing!!!!

I have this code example:
```
-- this is an example, for God's sake, that the final version is not like that.
local sg = server("3000") -- tic function

function TIC()
    sg:update()

    cls(0)
    
    print(sg:get("value"), 0, 0, 12)
end
```

and I also have this screenshot I made:
![multiplayer](https://user-images.githubusercontent.com/92587199/141666193-db353640-ca42-4cba-8a11-503d694f9a7b.png)

what it is?
![multiplayer_server](https://user-images.githubusercontent.com/92587199/141666417-741b5e14-fcae-4dd1-9bd1-07cbbb9fde8a.png)

**Comment by nesbox** [comment]
Great, also pls look at https://github.com/nesbox/TIC-80/issues/457

**Reply by bettywolfy** [reply]
ok.
